src/Response.php

```diff
@@ -428,12 +526,48 @@ protected function appendHeaders(): self
 
         // Set application headers
         foreach ($this->headers as $key => $value) {
-            \header($key . ': ' . $value);
+            $this->sendHeader($key, $value);
         }
 
         return $this;
     }
 
+    /**
+     * Send Header
+     * 
+     * Output Header
+     * 
+     * @param string $key
+     * @param string $value
+     * 
+     * @return void
+     * 
+     */
+    protected function sendHeader(string $key, string $value): void
+    {
+        \header($key . ': ' . $value);
+    }
+
+    /**
+     * Send Cookie
+     * 
+     * Output Cookie
+     * 
+     * @param string $name
+     * @param string $value
+     * @param array $options
+     * 
+     * @return void
+     */
+    protected function sendCookie(string $name, string $value, array $options)
+    {
+        if (\version_compare(PHP_VERSION, '7.3.0', '<')) {
+            \setcookie($name, $value, $options['expire'], $options['path'], $options['domain'], $options['secure'], $options['httponly']);
```

Comment:
We should handle default values for these keys here to avoid warnings.

Reply:
don't understand what you mean

Reply:
Instead of directly accessing `$options['expire']` and assuming it is set, we should have something like: `$expire = $options['expire'] ?? 0;`